index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <title>Apple</title>
+</head>
+<body>
+    <main>
+        <header>
+            <nav>
+                <ul class="clearfix">
+                    <li><img src="./assets/img/apple-logo.png" alt="Apple Logo"></li>
+                    <li>Store</li>
+                    <li>Mac</li>
+                    <li>iPod</li>
+                    <li>iPhone</li>
+                    <li>iPad</li>
+                    <li>iTunes</li>
+                    <li>Support</li>
+                    <li>
+                        <form action="#">
+                            <input type="search"/>
+                            <img src="./assets/img/search.png" alt="search icon">
+                        </form>
+                    </li>
+                </ul>
+            </nav>
+        </header>
+
+        <section class="intro">
+            <h1>What will your verse be?</h1>
+            <h2>Everyone has something to share. See how people are using iPad in their unique ways to add to the world's story.</h2>
+            <h3>Whatch the "Your Verse" film and explore the stories</h3>
+        </section>
+
+        <div class="featured">
+            <div><img src="./assets/img/1.jpg" alt=""></div>
```

Comment:
**[REQUIRED]** This images should be links to the articles or `"#"`. Please change the parent of the `image` element from `div` to `img`